### ISSUE TITLE
Check for TESTDIR needs to include previous named repo

### ISSUE THREAD

**Issue by smlambert** [adoptium/aqa-tests#2682]
The code in get.sh that checks if TESTDIR exists (see [get.sh#L157-L161](https://github.com/adoptium/aqa-tests/blob/master/get.sh#L157-L161)) works well for those who have forked the adoptium/aqa-tests repo.  It does not work for those who have forked prior to the renaming of aqa-tests (from openjdk-tests repo).   [Lots of members](https://github.com/adoptium/aqa-tests/network/members) with forks of openjdk-tests vs aqa-tests would be affected if they try to setup and run locally.

It should be updated to accept either test directory name.  Also noting that when I try to bypass this by passing in -t SomeDirName (or --testdir SomeDirName) where i have cloned this repo to, it also does not work.  if someone does pass in a test directory name via the -t option, it should be the value that is used to verify.

Or is there a better approach to this issue?

**Comment by sophia-guo** [comment]
I'm not sure if we still need the parameter `--testdir`. Most of functions have been moved out the initial get.sh file ( TKG, dependent lib setup, etc.) current requirement is TKG has to be under the same directory of get.sh or aqa-test(openjdk-tests) root directory no matter what the directory name is. The default value `TESTDIR="$(pwd)` should work good.

**Comment by smlambert** [comment]
I agree, we did not originally have it... I looked to see history of how we came to have the check:
https://github.com/adoptium/aqa-tests/pull/1646
introduced it.  Presumably its to support a case where someone wants to put test material elsewhere, but that not be preferred (or needed).  

I can close this and we can create a new "good first issue" to remove the check, or we can just reuse this one.  We still need 15 more good first issues for Grace Hopper open-source day in July.

**Comment by llxia** [comment]
In the test job, we run `get.sh` from the parent dir (not directly under aqa-tests) (see below). In that case, `--testdir|-t ` (TESTDIR) has to be set as the default `$(pwd)` value is not correct.

```
[2021-06-28T18:24:37.887Z] + ./aqa-tests/get.sh -s /home/jenkins/workspace/Grinder -t /home/jenkins/workspace/Grinder/aqa-tests -p x86-64_linux -r nightly -j 8 -i openj9 --openj9_repo https://github.com/eclipse/openj9.git --openj9_branch master --tkg_repo https://github.com/AdoptOpenJDK/TKG.git --tkg_branch master
```
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/997

If we want to remove `--testdir|-t `, then we need to update test jobs to run `.get.sh` directly under `aqa-tests`.  And https://github.com/adoptium/aqa-tests/blob/master/get.sh#L158 is to ensure TESTDIR points at aqa-tests dir (especially for local runs). 

In summary, we could have two issues here:
1. remove the check https://github.com/adoptium/aqa-tests/blob/master/get.sh#L158
2. remove `--testdir|-t ` in get.sh, update jenkinsfilebase to run `.get.sh` directly under `aqa-tests` and do not pass in TESTDIR

**Comment by smlambert** [comment]
Yes, let's spin this out into the 2 separate issues then, thanks for clarifying @llxia !

**Comment by llxia** [comment]
https://github.com/adoptium/aqa-tests/issues/2690 and https://github.com/adoptium/aqa-tests/issues/2691 are created.

**Comment by smlambert** [comment]
Thanks @llxia - I will close this one in favour of the 2 #2690 and #2691.
